desktop_files/package.json

```diff
@@ -55,7 +55,7 @@
     "querystring-es3": "0.2.1",
     "react": "16.4.1",
     "react-dom": "16.4.2",
-    "react-native": "git+https://github.com/status-im/react-native-desktop.git#master",
+    "react-native": "git+https://github.com/status-im/react-native-desktop.git#windows",
```

Comment:
I believe this shouldn't be commited.

Reply:
Yes, this is something that will be removed before merging this PR. Currently needed to test the Jenkins build.

Reply:
Yes, but we can only do that together with this PR, since as we saw in Prague that commit breaks status-react develop for some reason.

Reply:
You can create the PR now, to make it reviewed. And then commit whenever it is appropriate.
Also since this commit - https://github.com/status-im/status-react/commit/ae3029c2d9931de50bd72a43f34ef21cf6d85866 status-react depends not on `master` branch, but on tagged on, so now you can freely commit changes in react-native-desktop, they won't propagate to status-react immediately

Reply:
I will commit it to master directly then, since it had already been reviewed and later reverted so as to not break status-react.